From e2ac32709b10d9c1be8f3d5a760f58026525b3c0 Mon Sep 17 00:00:00 2001
From: Valentin Raevsky <valentin@compulab.co.il>
Date: Wed, 26 Oct 2022 16:37:38 +0300
Subject: [PATCH] mach-imx: Suppress TMU temperature message

Signed-off-by: Valentin Raevsky <valentin@compulab.co.il>
---
 arch/arm/mach-imx/cpu.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm/mach-imx/cpu.c b/arch/arm/mach-imx/cpu.c
index 7211fcae72..a96b2c3d30 100644
--- a/arch/arm/mach-imx/cpu.c
+++ b/arch/arm/mach-imx/cpu.c
@@ -231,6 +231,7 @@ int print_cpuinfo(void)
 		mxc_get_clock(MXC_ARM_CLK) / 1000000);
 #endif
 
+#ifdef ENABLE_TMU_MESSAGE
 #if defined(CONFIG_IMX_THERMAL) || defined(CONFIG_IMX_TMU)
 	puts("CPU:   ");
 	switch (get_cpu_temp_grade(&minc, &maxc)) {
@@ -261,6 +262,7 @@ int print_cpuinfo(void)
 	}
 	puts("\n");
 #endif
+#endif
 
 #if defined(CONFIG_DBG_MONITOR)
 	if (readl(&dbg->snvs_addr))
-- 
2.17.1

