From eabce6adf684f2739969b7b2e387863462b36462 Mon Sep 17 00:00:00 2001
From: Valentin Raevsky <valentin@compulab.co.il>
Date: Mon, 27 Feb 2023 18:43:06 +0200
Subject: [PATCH] iot-gate-imx8plus: Enable phy address discovery logics

Signed-off-by: Valentin Raevsky <valentin@compulab.co.il>
---
 include/configs/iot-gate-imx8plus.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/include/configs/iot-gate-imx8plus.h b/include/configs/iot-gate-imx8plus.h
index 5dc357a57e..43fab35181 100644
--- a/include/configs/iot-gate-imx8plus.h
+++ b/include/configs/iot-gate-imx8plus.h
@@ -15,10 +15,12 @@
 #define CONFIG_ETHPRIME                 "eth0" /* Set eqos to primary since we use its MDIO */
 
 #define CONFIG_FEC_XCV_TYPE             RGMII
+#define CONFIG_FEC_MXC_PHYADDR         	-1
 #define FEC_QUIRK_ENET_MAC
 
 #ifdef CONFIG_DWC_ETH_QOS
 #define CONFIG_SYS_NONCACHED_MEMORY     (1 * SZ_1M)     /* 1M */
+#define DWC_NET_PHYADDR	               	-1
 #endif
 
 #define PHY_ANEG_TIMEOUT 20000
-- 
2.17.1

