From a8f3d7f86a49d0e0d1a8ff7d8a333087cdb1d078 Mon Sep 17 00:00:00 2001
From: Valentin Raevsky <valentin@compulab.co.il>
Date: Sun, 10 Jul 2022 12:15:26 +0300
Subject: [PATCH] imx8mp: compulab: Enable binman

Signed-off-by: Valentin Raevsky <valentin@compulab.co.il>
---
 arch/arm/mach-imx/imx8m/Kconfig | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/arch/arm/mach-imx/imx8m/Kconfig b/arch/arm/mach-imx/imx8m/Kconfig
index 18483e9a14..0ee7d68bce 100644
--- a/arch/arm/mach-imx/imx8m/Kconfig
+++ b/arch/arm/mach-imx/imx8m/Kconfig
@@ -317,21 +317,27 @@ config TARGET_IOT_GATE_IMX8
 
 config TARGET_UCM_IMX8M_PLUS
 	bool "CompuLab ucm-imx8m-plus"
+	select BINMAN
 	select IMX8MP
 	select SUPPORT_SPL
 	select IMX8M_LPDDR4
+	select SUPPORT_EXTENSION_SCAN
 
 config TARGET_SOM_IMX8M_PLUS
 	bool "CompuLab som-imx8m-plus"
+	select BINMAN
 	select IMX8MP
 	select SUPPORT_SPL
 	select IMX8M_LPDDR4
+	select SUPPORT_EXTENSION_SCAN
 
 config TARGET_IOT_GATE_IMX8PLUS
 	bool "CompuLab iot-gate-imx8plus"
+	select BINMAN
 	select IMX8MP
 	select SUPPORT_SPL
 	select IMX8M_LPDDR4
+	select SUPPORT_EXTENSION_SCAN
 
 endchoice
 
-- 
2.17.1

