From db983e82192f2cfbfe7a37e94ac3112ba8fea757 Mon Sep 17 00:00:00 2001
From: Valentin Raevsky <valentin@compulab.co.il>
Date: Wed, 1 Mar 2023 18:01:23 +0200
Subject: [PATCH] arm64: config: Add compulab.config

Signed-off-by: Valentin Raevsky <valentin@compulab.co.il>
---
 arch/arm64/configs/compulab.config | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/compulab.config b/arch/arm64/configs/compulab.config
index 0c3eed846dc8..3f175dace15c 100644
--- a/arch/arm64/configs/compulab.config
+++ b/arch/arm64/configs/compulab.config
@@ -1,3 +1,4 @@
 CONFIG_EXPERT=y
 CONFIG_GPIO_SYSFS=y
 CONFIG_DEBUG_KERNEL=y
+CONFIG_FB_EFI=n
-- 
2.17.1

