From 0576462283aaec857af116c1d9aa5a9283b85a80 Mon Sep 17 00:00:00 2001
From: Valentin Raevsky <valentin@compulab.co.il>
Date: Thu, 3 Jun 2021 15:22:52 +0300
Subject: [PATCH] media: i2c: imx219: Make it work with imx8mp

Signed-off-by: Valentin Raevsky <valentin@compulab.co.il>
---
 drivers/media/i2c/imx219.c | 25 +++++++++++++++++++++++++
 1 file changed, 25 insertions(+)

diff --git a/drivers/media/i2c/imx219.c b/drivers/media/i2c/imx219.c
index 0ae66091a696..7ab50a44cc2a 100644
--- a/drivers/media/i2c/imx219.c
+++ b/drivers/media/i2c/imx219.c
@@ -1231,7 +1231,13 @@ static int imx219_identify_module(struct imx219 *imx219)
 	return 0;
 }
 
+static int v4l2_s_power(struct v4l2_subdev *sd, int on)
+{
+	return 0;
+
+}
 static const struct v4l2_subdev_core_ops imx219_core_ops = {
+	.s_power	= v4l2_s_power,
 	.subscribe_event = v4l2_ctrl_subdev_subscribe_event,
 	.unsubscribe_event = v4l2_event_subdev_unsubscribe,
 };
@@ -1421,6 +1427,24 @@ static int imx219_check_hwcfg(struct device *dev)
 	return ret;
 }
 
+/**
+ * camera_link_setup
+ *
+ * Note: Function is needed by imx8mp
+ *
+ * Returns always zero
+ */
+static int imx219_camera_link_setup(struct media_entity *entity,
+				    const struct media_pad *local,
+				    const struct media_pad *remote, u32 flags)
+{
+	return 0;
+}
+
+static const struct media_entity_operations imx219_camera_sd_media_ops = {
+	.link_setup = imx219_camera_link_setup,
+};
+
 static int imx219_probe(struct i2c_client *client)
 {
 	struct device *dev = &client->dev;
@@ -1501,6 +1525,7 @@ static int imx219_probe(struct i2c_client *client)
 	imx219->sd.internal_ops = &imx219_internal_ops;
 	imx219->sd.flags |= V4L2_SUBDEV_FL_HAS_DEVNODE;
 	imx219->sd.entity.function = MEDIA_ENT_F_CAM_SENSOR;
+	imx219->sd.entity.ops = &imx219_camera_sd_media_ops;
 
 	/* Initialize source pad */
 	imx219->pad.flags = MEDIA_PAD_FL_SOURCE;
-- 
2.11.0

