From 7c77c6c7ba11638e53fafc3bd5baf4e0e48b42b5 Mon Sep 17 00:00:00 2001
From: Valentin Raevsky <valentin@compulab.co.il>
Date: Tue, 3 Aug 2021 17:27:06 +0300
Subject: [PATCH] uclass: debug 4

Signed-off-by: Valentin Raevsky <valentin@compulab.co.il>
---
 drivers/core/uclass.c | 7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

diff --git a/drivers/core/uclass.c b/drivers/core/uclass.c
index f6c0705a16..2358e51aae 100644
--- a/drivers/core/uclass.c
+++ b/drivers/core/uclass.c
@@ -265,8 +265,7 @@ int uclass_find_next_device(struct udevice **devp)
 
 static int my_case=0;
 void set_case(int value) {
-	printf("%s [ my_case %d ]\n",__func__,my_case);
-
+	printf("%s [ my_case %d ]\n",__func__,value);
 	my_case = value;
 }
 
@@ -279,8 +278,10 @@ int uclass_find_device_by_name(enum uclass_id id, const char *name,
 
 	*devp = NULL;
 	if (my_case == 1) { printf("%s [ name %s ]\n",__func__,name); }
-	if (!name)
+	if (!name) {
+		printf("%s [ name %p ] EINVAL\n",__func__,name);
 		return -EINVAL;
+	}
 	ret = uclass_get(id, &uc);
 	if (ret)
 		return ret;
-- 
2.17.1

