From faa91a5fc3c82f671a854f138e6be0f351db7190 Mon Sep 17 00:00:00 2001
From: Valentin Raevsky <valentin@compulab.co.il>
Date: Tue, 2 Mar 2021 12:39:45 +0200
Subject: [PATCH] ucm-imx8m-plus: Update std/[out,err,in]

Signed-off-by: Valentin Raevsky <valentin@compulab.co.il>
---
 include/configs/compulab-imx8m-plus.h | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/include/configs/compulab-imx8m-plus.h b/include/configs/compulab-imx8m-plus.h
index 45bde3c691..e20f068391 100644
--- a/include/configs/compulab-imx8m-plus.h
+++ b/include/configs/compulab-imx8m-plus.h
@@ -79,6 +79,9 @@
 #define CONFIG_EXTRA_ENV_SETTINGS		\
 	CONFIG_MFG_ENV_SETTINGS \
 	JAILHOUSE_ENV \
+	"stdout=serial,vidconsole\0" \
+	"stderr=serial,vidconsole\0" \
+	"stdin=serial,usbkbd\0" \
 	"autoload=off\0" \
 	"script=boot.scr\0" \
 	"image=Image\0" \
-- 
2.11.0

