From b7002a5876a624cf2a340734f67868afd7c0f3b0 Mon Sep 17 00:00:00 2001
From: Valentin Raevsky <valentin@compulab.co.il>
Date: Wed, 3 Mar 2021 16:13:02 +0200
Subject: [PATCH] spi-nor-id: Add w25q64dw support

Signed-off-by: Valentin Raevsky <valentin@compulab.co.il>
---
 drivers/mtd/spi/spi-nor-ids.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/mtd/spi/spi-nor-ids.c b/drivers/mtd/spi/spi-nor-ids.c
index 1fd2adeeaf..3a3cdbc290 100644
--- a/drivers/mtd/spi/spi-nor-ids.c
+++ b/drivers/mtd/spi/spi-nor-ids.c
@@ -281,6 +281,7 @@ const struct flash_info spi_nor_ids[] = {
 			SPI_NOR_HAS_LOCK | SPI_NOR_HAS_TB)
 	},
 	{ INFO("w25x64", 0xef3017, 0, 64 * 1024, 128, SECT_4K) },
+	{ INFO("w25q64dw", 0xef6017, 0, 64 * 1024, 128, SECT_4K ) },
 	{
 		INFO("w25q64dw", 0xef6017, 0, 64 * 1024, 128,
 			SECT_4K | SPI_NOR_DUAL_READ | SPI_NOR_QUAD_READ |
-- 
2.11.0

